src/lib/pages/tx-details/components/tx-message/TxMsgDetails.tsx

```diff
@@ -21,7 +22,7 @@ export const TxMsgDetails = ({
   const receipts = generateReceipts(txMsgData, getAddressType, assetInfos)
     .concat(
       txMsgData.log && {
-        title: "Event Logs",
+        title: plur("Event Log", txMsgData.log.events.length),
```

Comment:
what happens if the provided length is undefined

Reply:
will show pluralize version of the text, but I think this is fine